### PR TITLE
Mimetype

### PR DESCRIPTION
**Pull request by Dletta**
Fixing the Mime-Type assignment in the express server. Workers are mimetype checked and will become strictly checked after COVID-19 is over and Mozilla puts in a new Security Policy.

Webworkers wouldn't work for me because of a path issue, but I discovered this issue during debugging and it make sense to serve js files with type "application/javascript"

### CODE REVIEW

**Comment by Dletta** [comment]
Sorry about the thousand commits during debug... lol

File: src/index.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
   <script src="https://cdn.jsdelivr.net/npm/gun/gun.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/gun/nts.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/gun/lib/unset.js"></script>
+  <script src="https://d3js.org/d3.v5.min.js" charset="utf-8"></script>
```
Comment (lmangani) [inline src/index.html:31]: this seems to be from another feature/pr? ;)

Comment (Dletta) [reply src/index.html:31]: I'll kick it out too, thanks for catching

**Comment by Dletta** [comment]
okay, url cleaner and removed the d3 link

**Comment by Dletta** [comment]
It works without it from assets/workers folder

**Pull request closed without merging**